src/js/components/put-away/PutAwayPage.jsx

```diff
@@ -369,10 +378,13 @@ class PutAwayPage extends Component {
    */
   // eslint-disable-next-line no-underscore-dangle
   filterMethod = (filter, row) => {
-    const rowData = row[filter.id];
-    const val = filter.id === 'inventoryItem.expirationDate'
-      ? this.props.formatLocalizedDate(rowData, DateFormat.COMMON)
-      : rowData;
+    let val = row[filter.id];
+    if (filter.id === 'inventoryItem.expirationDate') {
+      val = this.props.formatLocalizedDate(val, DateFormat.COMMON);
+    }
+    if (filter.id === 'product') {
+      val = val ? `${val.name} ${val.displayNameOrDefaultName}` : null;
+    }
```

Comment:
just of out curiosity - how does it work in the background? I can see such a filter method for the first time and it looks (at least) weird.

Reply:
it's a filter method that is build ion ReactTable. When we want to filter the table column we must check for the key (column accessory ) like
```if (filter.id === 'product')```
and return a boolean value if criteria matches the row value